ui/src/views/Tasks/TaskIndex/ListNamespaces/index.jsx

```diff
@@ -165,6 +165,7 @@ export default class ListNamespaces extends Component {
         error: taskNamespaceError,
       },
       description,
+      match: { params },
```

Comment:
This seems to throw an error for me locally. It turns out it was added on masters 2 days ago:

https://github.com/taskcluster/taskcluster/blob/266ecb3e97b857c324078eaf88a83034b456b232/ui/src/views/Tasks/TaskIndex/ListNamespaces/index.jsx#L157

Pulling the latest changes on master should fix this.